packages/app-harness/src/app/index.tsx

```diff
@@ -58,8 +82,8 @@ const App = () => {
                     <TestCase id={3} title="Orientation support ">
                         <OrientationLocker
                             orientation={PORTRAIT}
-                            onChange={(orientation) => console.log('onChange', orientation)}
-                            onDeviceChange={(orientation) => console.log('onDeviceChange', orientation)}
+                            onChange={(orientation) => logDebug(`onChange ${orientation}`)}
```

Comment:
wouldn't it be better if we refactor logDebug to use params?

Reply:
👌  done